tests/integration/test_design.py

```diff
@@ -2286,3 +2287,35 @@ def test_create_body_from_loft_profile(modeler: Modeler):
     # check volume of body
     # expected is 0 since it's not a closed surface
     assert result.volume.m == 0
+
+
+def test_revolve_sketch(modeler: Modeler):
+    """Test revolving a circular profile for a quarter donut."""
+    # Initialize the donut sketch design
+    design = modeler.create_design("quarter-donut")
+
+    # Donut parameters
+    path_radius = 5
+    profile_radius = 2
+
+    # Create the circlular profile on the XZ-plane centered at (5, 0, 0)
+    # with a radius of 2
+    plane_profile = Plane(
+        origin=Point3D([path_radius, 0, 0]), direction_x=UNITVECTOR3D_X, direction_y=UNITVECTOR3D_Z
+    )
+    profile = Sketch(plane=plane_profile)
+    profile.circle(Point2D([0, 0]), profile_radius)
+
+    # Revolver the profile around the Z-axis and centered in the absolute origin
```

Comment:
Should this be Revolve?

Reply:
Agh damn it. My bad... I will modify it. Good catch @PipKat!